### PR TITLE
feat: Improve UI with icon updates and font change

### PR DESCRIPTION
**Pull request by athuv**
## What type of PR is this?(check all applicable)
- [x] bug fix
- [x] optimization

### Related Issues

Issue #1079

### What this PR achieves

1. Corrected the map and share buttons icons.
2. Changed the font to Inter.
3. Implemented Nextjs font optimization (next/font).

### Screenshots, recordings
openbeta io_2

### CODE REVIEW

File: src/app/components/ShareAreaLinkButton.tsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ export const ShareAreaLinkButton: React.FC<{ uuid: string, areaName: string }> =
           setClicked(true)
         }}
       >
-        <LinkSimple size={20} /><span className='hidden md:inline'>Share</span>
+        <ShareNetwork  size={20} /><span className='hidden md:inline'>Share</span>
```
Comment (musoke) [inline src/app/components/ShareAreaLinkButton.tsx]: As discussed in #1079, I suggest leaving the icon as is and changing the text from "Share" to "Copy Link"

**Review by musoke — CHANGES_REQUESTED** [review]
Thank you for the PR!
I think your changes around the map and copy link buttons will be uncontroversial if you make the change I requested above.

Changing the font will likely need more discussion. In the interest of incorporating the icon changes faster, I suggest that you remove the font changes from this PR and submit them separately.

**Comment by vnugent** [comment]
@athuv can you fix linting errors and rebase? 
We use ts-standard which uses eslint under the hood.  You can install a VSC plugin:
https://marketplace.visualstudio.com/items?itemName=standard.vscode-standard

File: src/app/layout.tsx [hunk 2]
```diff
@@ -27,7 +28,7 @@ export default function RootLayout ({
 }): any {
   return (
     <html lang='en' className='snap-proximity snap-y'>
-      <body>
+      <body className={ inter.className}>
```
Comment (vnugent) [inline src/app/layout.tsx]: As discussed we will deal with font change in a separate task. In summary can you limit the PR to the following:
1. Update Map icon
2. Change share link label (not icon)

Thanks!

**Review by vnugent — CHANGES_REQUESTED** [review]

**Comment by athuv** [comment]
@vnugent @musoke 

I would like to apologize for the inconvenience caused by my recent changes. I understand that the font and icon changes were not aligned with the project's requirements. I have now reverted the changes as per your instructions. I appreciate your patience and understanding in this matter. I assure you that I will pay closer attention to the project guidelines in the future.

Thank you for your time and consideration.

**Comment by vnugent** [comment]
@athuv can you rebase, resolve conflicts and force push?  we're good to merge then.  Thanks!

**Comment by athuv** [comment]
@vnugent 
- [x] Rebase
- [x] Resolve conflicts 
- [x] force push

**Comment by vnugent** [comment]
@athuv can you redo the rebase?  Your recent commit included a lot of unwanted changes.  Since your PR is very small, it may be simpler to abandon the current working branch and repeat the work in a new PR.

**Pull request closed without merging**